validators/node-ssl.adoc

```diff
@@ -0,0 +1,140 @@
+= Node SSL Setup
+:icons: font
+:toc: macro
+
+ifdef::env-github[]
+
+:tip-caption: :bulb:
+:note-caption: :information_source:
+:important-caption: :heavy_exclamation_mark:
+:caution-caption: :fire:
+:warning-caption: :warning:
+
+endif::[]
+
+toc::[]
+
+== General info
+Node SSL certificates support secure communication with sChain endpoints. By default, each node of a SKALE Chain listens on HTTP and WS ports. If SSL certs exist on the node, then HTTPS and WSS ports are also turned on. 
+
+Node SSL certificates aren't required to start a node, register it, or create sChains. However, dApp developers using SKALE Chains need HTTPS/WSS endpoints to work with toolings such as API based wallets and other integrations. Therefore, SSL certificates are required for SKALE Network validator nodes.
```

Comment:
Does it make sense to mention that features like IMA require HTTPS as well?

Reply:
It's not required to use the IMA. It's just required from the customers. So resolving this comment